examples/nixnet/CAN_SIGNAL_SINGLE_POINT_OUTPUT.py

```diff
@@ -0,0 +1,87 @@
+
+import grpc
+import nixnet_pb2 as nixnet_types
+import nixnet_pb2_grpc as grpc_nixnet
+import getch
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+
+# Parameters
+INTERFACE = "CAN1"
+DATABASE = "NIXNET_example"
+CLUSTER = "CAN_Cluster"
+SIGNAL_LIST = "CANEventSignal1,CANEventSignal2"
+NUM_SIGNALS = 2
+SUCCESS = 0
+
+# Display Error Function
+def displayerror_andexit(Status, Source):
+    status_string = [None] * 1024
+    nixnet_types.StatustoString(Status,status_string.__sizeof__(),Source) 
+    print("\n\nERROR at {}!\n{}\n", Source, status_string)
+    print(Status)
+    print("\nExecution stopped.\nPress any key to quit\n")
+
+    client.Clear(nixnet_types.ClearRequest(session_ref = m_SessionRef))
+    exit(1)
+
+i = 0
+m_SessionRef = 0
+value_Buffer = [0] * NUM_SIGNALS
+
+# Create the communication channel for the remote host and create connections to the NI-DCPower and
+# session services.
+channel = grpc.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}")
+client = grpc_nixnet.NiXnetStub(channel)
+
+# Display parameters that will be used for the example.
+print("Interface: "+ INTERFACE, "Database: " + DATABASE, "Signal List: " + SIGNAL_LIST, sep="\n")
+
+# Create an XNET session in SignalOutSinglePoint mode
+CreateSession_Response = client.CreateSession(
+    nixnet_types.CreateSessionRequest(
+        database_name = DATABASE,
+        cluster_name = CLUSTER,
+        list = SIGNAL_LIST,
+        interface = INTERFACE, 
+        mode = nixnet_types.CREATE_SESSION_MODE_MODE_SIGNAL_OUT_SINGLE_POINT
+    ))
+
+m_SessionRef = CreateSession_Response.session_ref
+
+if CreateSession_Response.status == SUCCESS:
+    print("Session Created Successfully.\n")
+else:
+    displayerror_andexit(CreateSession_Response.status,"CreateSession")
+    
+print("\nPress any key to transmit new signal values or q to quit\n")
+
+while not (getch.getch()).decode('UTF-8') == 'q':
+    value_Buffer[0] = float(i)
+    value_Buffer[1] = float(i*10)
+    
+    # Update the signal data
+    response = client.WriteSignalSinglePoint(
+        nixnet_types.WriteSignalSinglePointRequest(
+            session_ref = m_SessionRef,
+            value_buffer = value_Buffer
+        ))
+    
+    if response.status == SUCCESS: 
+        print("Signals sent:")
+        print("Signal 1: ", value_Buffer[0])
```

Comment:
Here we can use `f` strings. Something like this `print(f"Signal 1: {value_buffer[0]}")`